wiki/de/de-Command-Line-Options.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: wiki
+title: "Kommandozeilen Optionen"
+lang: "en"
```

Comment:
Hi SmuSmu,

Thank you for your traslation! You need to change this to lang: "de". (Sonst wird die Seite als Englisch markiert )

Reply:
I'd suggest you to use a translation software like https://www.deepl.com/translator. The quality is quite good.